clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaStoreIngestionService.java

```diff
@@ -244,35 +242,26 @@ public KafkaStoreIngestionService(
       veniceWriterProperties.put(KAFKA_LINGER_MS, DEFAULT_KAFKA_LINGER_MS);
     }
 
+    // TODO: Once we start testing with other PubSub systems, we'll inject corresponding systems
```

Comment:
Checking here: My personal idea about this future work is leaving TODO is fine, but it would be great if we create internal ticket to track this.

Reply:
Done.